cmd/meroxa/root/environments/create.go

```diff
@@ -134,9 +133,21 @@ func (c *Create) Execute(ctx context.Context) error {
 		return err
 	}
 
-	c.logger.Infof(ctx, "Environment %q is being provisioned. Run `meroxa env describe %s` for status", environment.Name, environment.Name)
-	c.logger.JSON(ctx, environment)
+	if environment.Status.State == meroxa.EnvironmentStatePreflightError {
+		details := utils.EnvironmentPreflightTable(environment)
+		c.logger.Errorf(ctx,
+			"Environment %q could not be provisioned because it failed the preflight checks\n%s\n",
+			environment.Name,
+			details)
+	} else {
+		environment.Status.PreflightDetails = nil
```

Comment:
I don't think it's necessary to clear out preflight check details. Customers might want to see this info in the --json command
 ```suggestion
```